bindata/network/ovn-kubernetes/managed/ovnkube-node.yaml

```diff
@@ -83,6 +97,10 @@ spec:
             fi
             sleep 2
           done
+
+          {{if .ENABLE_OVN_NODE_PROXY}}
```

Comment:
Genuine question, does the container runtime not do that (kill all processes of a container when the container ends)?

Reply:
Not always. Being pid1 is special. I'm not sure if that is the case here, but there are definitely weird issues.

Specifically, pid1 doesn't get signals unless explicitly declared.